test/iterable_matchers_test.dart

```diff
@@ -199,6 +199,26 @@ void main() {
         "Which: has no match for a value greater than <3> at index 0");
   });
 
+  test('containsAll', () {
+    var d = [0, 1, 2];
+    shouldPass(d, containsAll([1, 2]));
+    shouldPass(d, containsAll([2, 1]));
+    shouldPass(d, containsAll([greaterThan(0), greaterThan(1)]));
+    shouldPass([2, 1], containsAll([greaterThan(0), greaterThan(1)]));
+    shouldFail(
+        d,
+        containsAll([1, 2, 3]),
+        "Expected: contains all of ([1, 2, 3]) "
+        "Actual: [0, 1, 2] "
+        "Which: has no match for <3> at index 2");
+    shouldFail(
+        1,
+        containsAll([1]),
+        "Expected: contains all of ([1]) "
+        "Actual: <1> "
+        "Which: not iterable");
```

Comment:
lets add a test for  the `expect([3], containsAll([greaterThan(0), greaterThan(1)]))` failure case.

Reply:
Oh sorry, the edge case I wanted to test was actually where both matchers match the same item. So I guess `expect([-1, 3], containsAll([greaterThan(0), greaterThan(1)]))` would be required.

Reply:
Done